stock_custom_exchange_rate_date/models/stock_move.py

```diff
@@ -0,0 +1,26 @@
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.multi
+    def _get_price_unit(self):
```

Comment:
BTW

@luisg123v @JulioSerna This method will only run when the Picking is validated? Won't it?

In case that the Pedimento is set as a date after the Picking is validated, what will be the approch?

Will the Journal Be modified, Is there a provision for it, or the picking will remain the same?


Regards

Reply:
Once the picking is validated, this field is readonly. Not sure if that answer your questiong, though, or you're asking for a new feature.

Reply:
@luisg123v You have nailed it.

So if the picking is validated prior to receiving the pedimento then odds are that the padimento date is different that the one in the picking.

How will we deal with this? @JulioSerna @rsilvam21 @nhomar 

Thanks